DEPS: Fix parallel build.  gmake recognizes libxml2.la and ./libxml2.la
as equivalent, ours does not.

Index: Makefile.am
--- Makefile.am.orig
+++ Makefile.am
@@ -112,7 +112,7 @@ if WITH_XPTR_SOURCES
 libxml2_la_SOURCES += xlink.c xpointer.c
 endif
 
-DEPS = $(top_builddir)/libxml2.la
+DEPS = libxml2.la
 LDADDS = $(top_builddir)/libxml2.la
 
 
